springfield/firefox/templates/firefox/browsers/mobile/ios.html

```diff
@@ -22,17 +22,20 @@
   {% endif %}
 {% endblock %}
 
-{# Issue 13019: Avoid duplicate content for English pages. #}
 {%- block page_title_full -%}
-  {%- if LANG == 'en-US' -%}
-    Get Firefox for iOS (iPhone and iPad) — Mozilla (US)
-  {%- elif LANG == 'en-GB' -%}
-    Download Firefox for iOS (iPhone and iPad) — Mozilla (UK)
+  {%- if LANG == 'en-US' or LANG == 'en-GB' -%}
+    Get Firefox for iOS (iPhone and iPad) from Mozilla
   {%- else -%}
     {{ ftl('mobile-ios-firefox-browser-ios') }} - Mozilla
```

Comment:
Here BTW the format for RoW seems to keep the old style though then. For not introducing a new l10n string just for potential "from Mozilla" here I guess?